src/resources/postcss/tickets-commerce/_checkout.pcss

```diff
@@ -311,6 +281,20 @@
 		}
 	}
 
+	.tribe-tickets__commerce-checkout-purchaser-info-wrapper {
+		margin-bottom: var(--tec-spacer-9);
+
+		label.tribe-tickets__commerce-checkout-purchaser-info-email-field-label,
+		label.tribe-tickets__commerce-checkout-purchaser-info-name-field-label {
+			font-size: var(--tec-font-size-2);
+		}
```

Comment:
I think we can rely on `tribe-common-b2` using that as the label class rather than defining it here. `tribe-common-b2` is using `--tec-font-size-2`. https://github.com/the-events-calendar/tribe-common-styles/blob/main/mixins/_typography.pcss#L37